pkg/controller/acmpca/certificateauthority/controller.go

```diff
@@ -54,11 +57,14 @@ const (
 )
 
 // SetupCertificateAuthority adds a controller that reconciles ACMPCA.
-func SetupCertificateAuthority(mgr ctrl.Manager, l logging.Logger) error {
+func SetupCertificateAuthority(mgr ctrl.Manager, l logging.Logger, limiter workqueue.RateLimiter) error {
```

Comment:
```suggestion
func SetupCertificateAuthority(mgr ctrl.Manager, l logging.Logger, global workqueue.RateLimiter) error {
```

I wonder if something like this that indicates what the rate limiter we're being passed would be worth it? Perhaps not - we don't do that for the `logging.Logger`. If we're going to stick with something generic I'd prefer something shorter - e.g. `rl workqueue.RateLimiter`. `limiter` seems redundant given it's always followed by the type, and given that the variable doesn't live very long lines-from-definition wise.

Reply:
I also wonder whether we're getting close to the point that we might want to change the `Setup` signatures (again) to accomodate more options like this - e.g. `SetupFoo(ctrl.Manager, setup.Option...)`. No need to bring that in scope here though.

Reply:
`rl` sounds good to me 👍